scripts/docker-run.sh

```diff
@@ -19,6 +19,8 @@ fi
 docker run -i -v "$(pwd)":/home/sof/work/sof.git \
 	--env CMAKE_BUILD_TYPE \
 	--env PRIVATE_KEY_OPTION \
-	   --user "$(id -u)" \
+	--env http_proxy=${http_proxy} \
+	--env https_proxy=${https_proxy} \
```

Comment:
Something tells me you haven't run shellcheck :-)

Reply:
shellcheck saves a live!